src/tlo/methods/breast_cancer.py

```diff
@@ -666,6 +668,8 @@ def apply(self, person_id, squeeze_factor):
 
         # Use a biopsy to diagnose whether the person has breast Cancer:
         # todo: request consumables needed for this
+        self.used_equipment = {'Slice Master sample processing Unit', 'Paraffin Dispense', 'Whatever used with biopsy',
```

Comment:
As the HSI event will already have `self.used_equipment` as an empty set. I think we should encourage users to use `self.used_equipment.update(['Slice Master sample processing Unit', 'Paraffin Dispense'])`

Reply:
I think a good style would be:
1) If the items to be used in the HSI are always the same (most cases), do the delcaration in `__init__` alongside other declarations: 
`self.equipment = {'item1', 'item2', etc.}` 
2) If the items may change during the HSI, then the declaration can be made/updated during the `apply` function:
`self.equipment = {'item1', 'item2', etc.}  # over-writes what was in the __init__` 
OR: `self.equipment.update(['item1', 'item2', etc.])  # add to what was in the __init__` 

In both cases, we tell the user that this isn't "read" until the HSI has run.

Reply:
Looks perfect.